regression-test/suites/query_p0/system/test_table_options.groovy

```diff
@@ -184,6 +177,41 @@ suite("test_table_options") {
         );
     """
 
-    qt_select """select * from information_schema.table_options where table_schema=\"${dbName}\" order by TABLE_NAME; """
-    sql "drop database if exists ${dbName}"
+    qt_select_check_1 """select * from information_schema.table_options where table_schema=\"${dbName}\" order by TABLE_NAME; """
+    sql """
+        drop table test_row_column_page_size2;
+    """    
+    qt_select_check_2 """select * from information_schema.table_options where table_schema=\"${dbName}\" order by TABLE_NAME; """    
```

Comment:
should order by all columns to make sure the case is stable

Reply:
thanks for the comment,  i have handled it.